SYCL/ESIMD/api/functional/common.hpp

```diff
@@ -55,6 +55,10 @@ using shared_allocator = sycl::usm_allocator<DataT, sycl::usm::alloc::shared>;
 template <typename DataT>
 using shared_vector = std::vector<DataT, shared_allocator<DataT>>;
 
+constexpr int ceil(int a, int b) {
```

Comment:
please add description

Reply:
BTW, here is the adopted definition of `ceil` function:
The ceil() function returns the smallest integer value that is greater than or equal to a number.

this `ceil` does something different, so please also change the name

Reply:
The my `ceil` function has been renamed and the description has been added in this commit: https://github.com/intel/llvm-test-suite/pull/931/commits/f78dac9bb1254cf151c0714dad7700561ac03640